blueocean/pom.xml

```diff
@@ -108,13 +108,6 @@
             <artifactId>blueocean-jira</artifactId>
         </dependency>
 
-        <!-- JENKINS-56773 - Disabled until executor plugin has performance improvements made
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>blueocean-executor-info</artifactId>
-        </dependency>
-        -->
-
```

Comment:
In order to tombstone it you need to do the exact opposite: uncomment this section (and revert the removal from root POM `dependencyManagement` below. Otherwise the tombstone will not get released and put on the update center! _After_ you do the next reactor release, including the tombstone, you can actually remove this entry _and_ delete the whole module from sources.

Reply:
arfff my bad

Reply:
This change reinstalled the deprecated plugin for me and now shows a warning in jenkins. So i hope there will be a new release soon, which removes / disables this again 😞

Reply:
Yes that is the idea.

The dependency from `blueocean` should have been optional I suppose. Anyway, the next release should remove the dep altogether.